cpp/src/examples/gcylon/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+##
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##
+
+add_definitions(-DEXAMPLE_CMAKE_DIR=\"${CMAKE_CURRENT_SOURCE_DIR}\")
+
+set(CMAKE_CUDA_COMPILER $ENV{CUDACXX})
+
+# project(GCYLON LANGUAGES CXX CUDA)
+enable_language(CUDA)
+
+set("GCYLON_SIMD_LEVEL" "SSE4_2") # options: SSE4_2| AVX512 | AVX2
```

Comment:
this sould come witin an if (NOT DEFINED CYLON_SIMD_LEVEL)

Reply:
added